include/filter-rest-routes.php

```diff
@@ -42,6 +42,10 @@ public function __construct( PLL_Model $model ) {
 
 		// Adds search REST endpoint.
 		$this->filtered_routes['search'] = 'wp/v2/search';
+
+		if ( $this->model->post_types->is_translated( 'wp_navigation' ) ) {
+			$this->filtered_routes['navigation-fallback'] = 'wp-block-editor/v1/navigation-fallback';
+		}
```

Comment:
`wp_navigation` is always translatable`.

Reply:
Not in Polylang Free I'm afraid.